### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by jwindhager**
Add the following fields:
- [x] `ExpectedRadios.no_fill`
- [x] `ExpectedRadios.no_outline`
- [x] `ExpectedHeader.stroke_width`

### CODE REVIEW

File: tissuumaps_schema/v01/project.py [hunk 1]
```diff
@@ -276,6 +276,10 @@ class ExpectedHeader(SchemaBaseModel):
             "Numerical value for a fixed opacity factor to be applied to markers."
         ),
     )
+    stroke_width: float = Field(
+        default=2.5,
+        description=("Numerical value for the marker stroke width."),
```
Comment (jwindhager) [inline tissuumaps_schema/v01/project.py]:
```suggestion
        description="Numerical value for the marker stroke width.",
```

File: tissuumaps_schema/v01/project.py [hunk 2]
```diff
@@ -345,9 +349,14 @@ class ExpectedRadios(SchemaBaseModel):
     )
     no_outline: bool = Field(
         default=False,
-        alias="_no_outline",
+        alias="no_outline",
```
Comment (jwindhager) [inline tissuumaps_schema/v01/project.py]:
```suggestion
```

File: tissuumaps_schema/v01/project.py [hunk 3]
```diff
@@ -345,9 +349,14 @@ class ExpectedRadios(SchemaBaseModel):
...
         description="If marker shapes should be rendered without outline.",
     )
+    no_fill: bool = Field(
+        default=False,
+        alias="no_fill",
```
Comment (jwindhager) [inline tissuumaps_schema/v01/project.py]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```
Comment (jwindhager) [inline CHANGELOG.md]:
```suggestion
## [v1.1.0](https://github.com/TissUUmaps/TissUUmaps-schema/compare/v1.0.2...v1.1.0)

### Added
- New field `ExpectedHeader.strock_width`
- New field: `ExpectedRadios.no_fill`

### Fixed

### Changed
- Renamed field `_no_outline` to `no_outline` (in JSON)

### Removed
```

**Comment by FredrikNysjo** [comment]
Accept pull request

**Review by FredrikNysjo — APPROVED** [review]

**Review by FredrikNysjo — APPROVED** [review]

File: tissuumaps_schema/v01/project.py (hunk 2 above)
Comment (jwindhager) [reply tissuumaps_schema/v01/project.py]: Discussed offline: do not rename, to not introduce breaking changes!

**Review by cavenel — APPROVED** [review]

**Pull request merged**